spec.html

```diff
@@ -903,13 +903,24 @@ <h1>Static Semantics</h1>
-      <p>The notation &ldquo;<emu-eqn id="eqn-modulo" aoid="modulo">_x_ modulo _y_</emu-eqn>&rdquo; (_y_ must be finite and nonzero) computes a value _k_ of the same sign as _y_ (or zero) such that <emu-eqn>abs(_k_) &lt; abs(_y_) and _x_-_k_ = _q_ &times; _y_</emu-eqn> for some integer _q_.</p>
-      <p>The mathematical function <emu-eqn id="eqn-floor" aoid="floor">floor(_x_)</emu-eqn> produces the largest integer (closest to positive infinity) that is not larger than _x_.</p>
+      <p>This specification makes reference to two kinds of numeric values:</p>
+      <ul>
+        <li><em>Number</em>: IEEE 754-2008 double-precision floating point values, used as the default numeric type.</li>
+        <li><em>Mathematical value</em>: Arbitrary real numbers, used for specific situations.</li>
+      </ul>
+
+      <p>In the language of this specification, numerical values and operations (including addition, subtraction, negation, multiplication, addition, and comparison) are distinguished among different numeric kinds using subscripts. The subscript <sub><dfn id="𝕗">𝕗</dfn></sub> refers to Numbers, and the subscript <sub><dfn id="𝕧">𝕧</dfn></sub> refers to numeric values. A subscript is used following each numeric value and operation. For brevity, the <sub>𝕗</sub> subscript can be omitted on Number values--a numeric value with no subscript is interpreted to be a Number. An operation with no subscript is interpreted to be a Number operation, unless one of the parameters has a particular subscript, in which case the operation adopts that subscript. For example, 1<sub>𝕧</sub> + 2<sub>𝕧</sub> = 3<sub>𝕧</sub> is a statement about mathematical values, and 1 + 2 = 3 is a statement about Numbers. It is not defined to mix Numbers and mathematical values in either arithmetic or comparison operations, and any such undefined operation would be an editorial error in this specification text.</p>
+      <p>The Number value 0, alternatively written 0<sub>𝕗</sub>, is defined as the double-precision floating point positive zero value. In certain contexts, it may also be written as +0 for clarity.</p>
+      <p>This specification denotes most numeric values in base 10; it also uses numeric values of the form 0x followed by digits 0-9 or A-F as base-16 values.</p>
+      <p>In certain contexts, an operation is specified which is generic between Numbers and mathematical values. In these cases, the subscript can be a variable; _t_ is often used for this purpose, for example 5<sub>_t_</sub> * 10<sub>_t_</sub> =<sub>_t_</sub> 50<sub>_t_</sub> for any _t_ ranging over 𝕧 and 𝕗.</p>
```

Comment:
It looks like you have t50t here, should the first one be removed?

Reply:
I think the first one is attached to the equals sign (i.e. equality comparison operation).

IMO this paragraph should state (probably in more elegant words) that the reason that example works is that 5, 10, and 50 are in the right range so as to not encounter the vagaries of floating point arithmetic.

Reply:
@domenic Yes, that was the intention.

Good suggestion, added a note explaining this reason.